setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
```

Comment:
This change isn't mentioned in the description, was this intentionally included?